File: python/stack.py
class Stack:
    '''
    Creating A Stack.

    Functions
    ---------
    push: add element to stack.
    pop: remove element from stack.
    print: prints itself.
    '''
    def __init__(self,*elements):
        self.list = list(elements)

    def push(self,*elements):
        for element in elements:
            self.list.append(element)

    def pop(self):
        return self.list.pop()

    def isempty(self):
        return not bool(len(self.list))

    def print(self):
        print(self.list)

# s1 = Stack()

# s1.push(5,2,4)
# s1.print()
# s1.pop()
# s1.print()

File: javascript/stack.js
class Stack{
	constructor(){
		this.array = []
	}
	push(ele){
		this.array.push(ele)
	}
	pop(){
		return this.array.pop()
	}
	print(){
		console.log(this.array)
	}
}

// var s = new Stack()
// s.push(10)
// s.print()
// s.push(198)
// s.print()
// s.push(2)
// s.push(53)
// s.print()
// s.pop()
// s.print()